### PR TITLE
added view column in photos table and modified the photos controller

### PR DESCRIPTION
**Pull request by KatherinaFatiguso**
Please review my codes

### CODE REVIEW

File: app/controllers/photos_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def find_album
 
   def find_photo
     @photo = Photo.find(params[:id])
+    @photo.view += 1;
```
Comment (jsscclr) [inline app/controllers/photos_controller.rb:11]: I think views might be a bit more understandable than view- view sounds a bit like a method name, so it's not as intuitive. Does this also save to the database? Alternatively you could use @photo.increment!(:views) :)

File: app/controllers/photos_controller.rb [hunk 2]
```diff
@@ -16,6 +17,11 @@ def index
 
   def edit;end
 
+  def show
+    @photo = Photo.find(params[:id])
```
Comment (jsscclr) [inline app/controllers/photos_controller.rb:21]: It doesn't look like you're calling find_photo- you can DRY up your code a little if you call it, I think!

File: app/controllers/photos_controller.rb (hunk 1 above)
Comment (kathf) [inline app/controllers/photos_controller.rb]: Another option would have been to use:  @photo.increment!(:view)

Comment (jsscclr) [inline app/controllers/photos_controller.rb:11]: Also, do you want the view to be incremented each time the photo is found? I think sometimes the photo object might be found but not necessarily used (like if you wanted to link it or only display a thumbnail, something like that).

File: app/controllers/photos_controller.rb [hunk 3]
```diff
@@ -16,6 +17,11 @@ def index
...
+    @photo.view += 1;
```
Comment (kathf) [inline app/controllers/photos_controller.rb]: Another option would have been to use: @photo.increment!(:view)

File: app/controllers/photos_controller.rb (hunk 1 above)
Comment (KatherinaFatiguso) [inline app/controllers/photos_controller.rb:11]: yes, I just realised that when I checked my notes. I should have used the increment! method, not the counter =+. 
Thank you

File: app/controllers/photos_controller.rb (hunk 2 above)
Comment (KatherinaFatiguso) [inline app/controllers/photos_controller.rb:21]: yes, I need to DRY up this codes, too many repetitions.
Thank you.

File: app/controllers/photos_controller.rb (hunk 1 above)
Comment (kathf) [inline app/controllers/photos_controller.rb:11]: I think that including this method here will increment the view count by one every time you call a new, create or index method in the photos controller (so a newly added photo will already have 2 views). So I think you can delete this line, and just leave the method on line 22.

Comment (KatherinaFatiguso) [inline app/controllers/photos_controller.rb:11]: maybe the view should not be incremented each time the photo is found, but the only place I can think of the place is in show controller

**Pull request merged**